mods/braindance_protocol/cheats/teleport.lua

```diff
@@ -153,6 +153,13 @@ function Teleport.GenericTheAtlantis()
     Utilities.FinishProtocol(moduleName)
 end
 
+function Teleport.GenericNomadStartLocationAWACSZone()
+    local moduleName = "Teleporting to Nomad Starting Location - Awacs Zone"
```

Comment:
Can this be changed to `AWACS`

Reply:
Yes, will do

Reply:
Oh wait a minute, do you mean the name of the function or the actual location?